sdk/src/driftClient.ts

```diff
@@ -6226,6 +6237,17 @@ export class DriftClient {
 			}
 		}
 
+		if (isUpdateHighLeverageMode(orderParams.bitFlags)) {
+			remainingAccounts.push({
+				pubkey: getHighLeverageModeConfigPublicKey(
+					this.program.programId,
+				),
+				isWritable: true,
+				isSigner: false,
+			});
+		}
+
```

Comment:
think you want to do this before maker/referrer infos since you consume HLM acc first in the ix